main/admin/course_fields.php

```diff
@@ -0,0 +1,275 @@
+<?php
```

Comment:
You don't need to create this file you can use just:

main/admin/extra_fields.php?type=course

once the "course" type was well defined in the "extra_fields" classes.